src/main/java/seedu/sellsavvy/logic/commands/ordercommands/MarkOrderCommand.java

```diff
@@ -59,13 +58,13 @@ public CommandResult execute(Model model) throws CommandException {
         }
 
         Order orderToMark = filteredOrderList.get(index.getZeroBased());
+        if (orderToMark.getStatus() == Status.COMPLETED) {
```

Comment:
I wonder if we should provide a Status.isCompleted() method to further OOP order status?

Reply:
Sure